src/send/Send.tsx

```diff
@@ -60,7 +65,11 @@ function Send({ route }: Props) {
   const allRecipients = useSelector(phoneRecipientCacheSelector)
   const recentRecipients = useSelector((state) => state.send.recentRecipients)
 
-  const tokensWithBalance = useSelector(tokensWithTokenBalanceAndAddressSelector)
+  const supportedNetworkIds = getSupportedNetworkIdsForSend()
+  const tokensWithBalance = useTokensWithTokenBalance()
```

Comment:
looks like useTokensWithTokenBalance filters by the network id's that are supported for token balances first. then you apply a filter here by network id's that are supported for send. I don't think that's how our feature flags should work-- seems like we should be able to turn on/off different multichain features independently of each other, and not have a hardcoded dependency where the send feature flag depends on the balances feature flag

Reply:
what about a hook like `useTokensForSend` that just uses the send feature flag, instead of a combination of both?

Reply:
I think that makes sense, in the case where supportedNetworks for send was `celo-mainnet` & `ethereum-mainnet`, but `tokensWithBalance` only included tokens from `celo-mainnet` it would be fine because only tokens on `celo-mainnet` would have balances available for send. For requests Celo stables are only available so there should be no impact.

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/4242/commits/775caebe434df7f28a1c32807065ccfe5e835413!